### ISSUE TITLE
Adding presets to windows 10 standalone

### ISSUE THREAD

**Issue by bebop210** [projectM-visualizer/projectm#267]
I find it impossible to add presets to my windows 10 standalone. I’m looking for presets and eyetune and sperl and the GD Rosetta Stone but windows 10 app architecture makes me wanna go die/cry/lie/fry/but definitely not try anymore... where can I deposit my thousands of presets and make this the show pony it’s crying out to be.

**Comment by coast77777777777** [comment]
I am curious, can you elaborate?
look here: https://github.com/projectM-visualizer/projectm/blob/916cd5966236d8c87bafbc97df5f4f66b7c379af/src/EyeTune/EyeTune.WindowsUniversal.Application/EyeTune.WindowsUniversal.Application.vcxproj#L174

add your milkdrops to /preset_eyetune
make a pull request

I will review them, then we merge your creations into xbox/pc app on windows app store

**Comment by reenenlaurie** [comment]
So I found some of my presets in the experimental folder, but there is no real way to traverse the 42000 presets in there and even if I find them eventually, and there is also no way to move my presets to the "vetted" presets.  Not even admin mode in windows.  (I am intrigued that I couldn't move it even after finding the installation)

**Comment by bebop210** [comment]
This is my experience too.. apparently one needs “special permission” to do anything with any file or any folder associated with any  app downloaded from windows store and every way I know to acquire this  
Is greyed out.  This has made adding any presets to the ones in the “p” folder impossible as far as I can tell.. other than feeling great about going full linux in my own pc’s and laptops, this is killing my abilities to show full supremacy of visual stimuli at my work place and I want it ended right this instant, jk but it is a really frustrating situation.

**Comment by ezibit** [comment]
Agreed...  a Preset folder for Windows 10 app to add the 41000 .milk files would be greatly appreciated... Major drawback!
